boards/others/esp32c3_supermini/esp32c3_supermini-pinctrl.dtsi

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Arrel Neumiller
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32c3-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32c3-gpio-sigmap.h>
+
+&pinctrl {
+
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_GPIO21>;
+			output-high;
+		};
+		group2 {
+			pinmux = <UART0_RX_GPIO20>;
+			bias-pull-up;
+		};
+	};
+
+	spim2_default: spim2_default {
+		group1 {
+			pinmux = <SPIM2_MISO_GPIO5>,
+				 <SPIM2_SCLK_GPIO4>,
+				 <SPIM2_CSEL_GPIO7>;
+		};
+		group2 {
+			pinmux = <SPIM2_MOSI_GPIO6>;
+			output-low;
+		};
+	};
+
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <I2C0_SDA_GPIO8>,
```

Comment:
I guess something is wrong withi the pinouts.
a) It states here that I2C is attached to GPIO8/9, but in .dts file, LED and GPIO are attached to GPIO8/9.
b) In documentation file, it states that I2C is attached to GPIO8/10.

1) I think you cane remove the I2C entry regarding its pinouts.
2) Fix the GPIO/LED/I2C pin accordingly in the .dtsi/.dts file.

Reply:
Will fix. Thanks for catching it.